src/modules/developer_tools.json

```diff
@@ -0,0 +1,250 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "DeveloperTools",
+        "description": "A module for supporting off-device developer tools.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "find",
+            "summary": "Starts a long-running DNS-SD search for a list of developer tools that match the standard 'Firebolt Developer Tool' service type",
+            "tags": [],
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "stopFind",
```

Comment:
I think `cancelFind` (or `cancelScan`) is a bit more appropriate, as it's a request being filed, that might be called at the same time a result is being discovered.

Also, I'm tempted to suggest just `cancel`.

I'm even wondering if we want to have a new method template/tag called `cancellable` where the SDK would generate the bindings to do something like:

```javascript
import { DeveloperTools } from '@firebolt-js/manage-sdk'

const request = DeveloperTools.scan() // this method creates a correlationId, calls the scan RPC method, and returns an object with a cancel() method

cancelButton.onclick = () => {
  request.cancel() // this method calls the RPC `cancelScan` method w/ a correlationId
}
```

Reply:
Up to you if you think this is a general pattern you think is worth doing. I haven't heard of other use cases that need it. For now, I've renamed to 'cancelScan' (I think 'cancel' is too ambiguous, personally.)